resources/views/layouts/default.blade.php

```diff
@@ -65,15 +65,8 @@
     <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
     <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
     <!--[if lt IE 9]>
-
-    @if ($snipeSettings->load_remote=='1')
         <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js" integrity="sha384-qFIkRsVO/J5orlMvxK1sgAt2FXT67og+NyFTITYzvbIP1IJavVEKZM7YWczXkwpB" crossorigin="anonymous"></script>
```

Comment:
But I thought the whole idea was to not load any JS externally anymore, because it's a business critical application. I don't have strong feelings either way, it just seems counter to your earlier argument for keeping everything local.

Reply:
I know it seems odd  :) I was really referring to the other files. Since IE 8 is such a small share it won't hardly ever be called. In my thinking the critical js files should be loaded locally. I can put it back to load locally doesn't really make a difference to me for those files.